pkg/skr/gcpnfsvolumerestore/checkRestoreOperation.go

```diff
@@ -19,7 +19,7 @@ func checkRestoreOperation(ctx context.Context, st composed.State) (error, conte
 
 	restore := state.ObjAsGcpNfsVolumeRestore()
 	opName := restore.Status.OpIdentifier
-	logger.WithValues("Nfs Restore source:", restore.Spec.Source.Backup.ToNamespacedName(state.Obj().GetNamespace()),
+	logger.WithValues("nfsRestoreSource:", restore.Spec.Source.Backup.ToNamespacedName(state.Obj().GetNamespace()),
```

Comment:
```suggestion
	logger.WithValues("nfsRestoreSource", restore.Spec.Source.Backup.ToNamespacedName(state.Obj().GetNamespace()),
```
Please remove the `:` at the end, as it is added automatically in the log aggregator.
`logger.Withvalues("foo", "bar")` will display `foo: bar`
`logger.WithValues("foo:", "bar")` will display `foo:: bar`